src/ui/MessageInput/utils.js

```diff
@@ -38,4 +38,10 @@ export const nodeListToArray = (childNodes) => {
   }
 };
 
-export default debounce;
+export function isChannelTypeSupportsMultipleFilesMessage(channel) {
+  return channel
+    && typeof channel.isGroupChannel === 'function'
+    && channel.isGroupChannel()
```

Comment:
nit; 
I think those two lines can be simplified like below. 
```suggestion
    && channel?.isGroupChannel?.()
```

Reply:
oh thanks! `&& channel?.isGroupChannel?()` I tried this and didnt work but now I understand `.` was missing.